### PR TITLE
Added logger for dao

### PR DESCRIPTION
**Pull request by aleksandr-hrankin**

### CODE REVIEW

File: src/main/java/ua/mate/cinema/Main.java [hunk 1]
```diff
@@ -38,6 +41,7 @@ public static void main(String[] args) throws AuthenticationException {
         bob.setPassword("1234");
         User authBob = authService.register(bob.getEmail(), bob.getPassword());
 
+        log.info("User auth successfully");
         System.out.println("auth successful: " + authService.login(authBob.getEmail(), "1234"));
         System.out.println("user shopping cart empty: " + cartService.getByUser(authBob));
```
Comment (KostiantynVorobiov) [inline src/main/java/ua/mate/cinema/Main.java]: Я так понял тут все эти System-и нужно поменять на log

File: src/main/java/ua/mate/cinema/dao/impl/OrderDaoImpl.java [hunk 2]
```diff
@@ -22,6 +25,7 @@ public Order add(Order order) {
             transaction = session.beginTransaction();
             session.persist(order);
             transaction.commit();
+            log.info("order added successfully");
```
Comment (KostiantynVorobiov) [inline src/main/java/ua/mate/cinema/dao/impl/OrderDaoImpl.java]:
```suggestion
            log.info("Оrder added successfully");
```

**Review by KostiantynVorobiov — APPROVED** [review]
ОК!

File: src/main/java/ua/mate/cinema/Main.java (hunk 1 above)
Comment (aleksandr-hrankin) [reply src/main/java/ua/mate/cinema/Main.java]: Точно, спасибо

File: src/main/resources/log4j.properties [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+# Redirect log messages to console
+log4j.appender.stdout=org.apache.log4j.ConsoleAppender
+log4j.appender.stdout.Target=System.out
+log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
+log4j.appender.stdout.layout.ConversionPattern=%d{yyyy-MM-dd HH:mm:ss} %-5p %c{1}:%L - %m%n
+
+# Redirect log messages to a log file, support file rolling.
+log4j.appender.file=org.apache.log4j.RollingFileAppender
+log4j.appender.file.File=logs/cinema.log
```
Comment (KseniiaMakarova) [inline src/main/resources/log4j.properties:12]: does it work for the relative path? just curious

File: src/main/java/ua/mate/cinema/dao/impl/ShoppingCartDaoImpl.java [hunk 4]
```diff
@@ -21,6 +24,7 @@ public ShoppingCart add(ShoppingCart shoppingCart) {
             transaction = session.beginTransaction();
             session.saveOrUpdate(shoppingCart);
             transaction.commit();
+            log.info("Shopping cart added successfully");
```
Comment (KseniiaMakarova) [inline src/main/java/ua/mate/cinema/dao/impl/ShoppingCartDaoImpl.java:27]: let's add logging of update operation as well

File: src/main/java/ua/mate/cinema/Main.java (hunk 1 above)
Comment (KseniiaMakarova) [reply src/main/java/ua/mate/cinema/Main.java]: I agree, please replace all souts
also, let's wrap `login()` call in try-catch and add log.warn with the informative message in catch block

**Review by KseniiaMakarova — CHANGES_REQUESTED** [review]

File: src/main/resources/log4j.properties (hunk 3 above)
Comment (aleksandr-hrankin) [reply src/main/resources/log4j.properties:12]: image
It's work)

**Review by KseniiaMakarova — APPROVED** [review]

**Pull request merged**